### DISCUSSION TITLE
Communities: Record claiming / Multiple communities support

### DISCUSSION THREAD

**Discussion by lnielsen** [inveniosoftware/product-rdm#161] [Ideas]

**Comment by fvarrato** [comment]
A use case we would be very interested in is a workflow where community managers can propose records for inclusion in a community, and where authors or uploaders can explicitly accept or reject that inclusion. This would be particularly useful for institutional communities (such as university‑level ones), which often need better ways to identify and showcase Open Research Data outputs in order to understand and report on their impact.

The key idea would be a controlled, opt‑in mechanism rather than direct curation power: community managers would be able to list a limited number of records in a dedicated dashboard and send inclusion requests, but the record only joins the community once the owner has approved it. We are aware of the potential for spam or misuse, so any implementation would need safeguards (rate limits, opt‑out options, or similar measures) to avoid unsolicited or mass outreach.

From our perspective at EPFL, such a feature would nicely complement curation‑based communities (like our EPFL Zenodo community) by preserving author control while still enabling institutions to actively surface relevant records in a transparent and auditable way. We would be very happy to further discuss the idea and its implications if this aligns with the roadmap or community priorities.
